metadata-transformer/src/main/java/no/unit/nva/metadata/extractors/AbstractExtractor.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.metadata.extractors;
+
+import no.unit.nva.model.EntityDescription;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.function.Function;
+
+public final class AbstractExtractor {
+
+    public static final Function<ExtractionPair, EntityDescription> apply = AbstractExtractor::extract;
+
+    @JacocoGenerated
```

Comment:
code coverage "cheating" or a good reason for this?

Reply:
Without the constructor, Jacoco claims the class is uncovered because the default constructor is uncovered

Reply:
I do not think that @JacocoGenerated is necessary. Jacoco ignores these constructors